ext/search_kit/Civi/Api4/Action/SearchDisplay/GetSearchTasks.php

```diff
@@ -17,7 +17,7 @@ class GetSearchTasks extends \Civi\Api4\Generic\AbstractAction {
 
   /**
    * An array containing the searchDisplay definition
-   * @var array
+   * @var string|array
```

Comment:
So this is only called from one place which you recently added - so the option of passing in as an array there is also viable. I will let @colemanw take a look & see if he has an opinion here

![image](https://user-images.githubusercontent.com/336308/210015583-71ebf533-8ce5-4e20-8a6b-15f7f2b5e60a.png)

Reply:
It looks like this is the one line needed for the fix, and the other change to `civiimport_civicrm_searchKitTasks` is more cosmetic. Rather than get bogged down in discussion of that, @mattwire could you please open a PR against the RC branch with just this one-line change?

Reply:
Resolved via #25482